kong/plugins/log-serializers/basic.lua

```diff
@@ -12,11 +12,12 @@ function _M.serialize(ngx)
       consumer_id = ngx.ctx.authenticated_credential.consumer_id
     }
   end
-  
+
   return {
     request = {
-      uri = ngx.var.request_uri,
-      request_uri = ngx.var.scheme .. "://" .. ngx.var.host .. ":" .. ngx.var.server_port .. ngx.var.request_uri,
+      request_uri = ngx.var.request_uri,
+      uri = ngx.var.uri,
+      request_url = ngx.var.scheme.."://"..ngx.var.host..":"..ngx.var.server_port..ngx.var.request_uri,
```

Comment:
so, this renames request_uri -> request_url, and makes request_uri a new value (the `$request_uri`) variable? this feels odd, and very much a breaking change. since we're introducing a new value `request_url`, why not just give it the new variable assignment?